.travis/template.txt

```diff
@@ -25,7 +25,7 @@ before_script:
 - cd ..
 script: 
 - cd ./.travis
-- bash -x -e ./build_package.sh $PACKAGE
+- if [ -n "$CGAL_DEBUG_TRAVIS" ]; then  bash -x -e ./build_package.sh $PACKAGE; else bash -e ./build_package.sh $PACKAGE; fi
```

Comment:
The Travis `CHECK` says that this template and the file `.travis.yml` do not agree:
https://travis-ci.org/CGAL/cgal/jobs/382181056#L1849